rtv/subreddit.py

```diff
@@ -108,10 +108,11 @@ def open_link(self):
         "Open a link with the webbrowser"
         data = self.content.get(self.nav.absolute_index)
 
-        if data['url'] == 'selfpost':
-            self.open_submission()
+        url = data['url_full']
+        if data['url'] != url:
```

Comment:
This isn't very descriptive as to what's happening. If a cross post's url was represented as simply `'x-post'` instead of `'x-post via /r/{sub}'`, you could do something like this.

```python
if data['url'] in ['selfpost', 'x-post']:
    ...
```

I think this would make things more explicit, and obvious as to what's actually happening.